syntax = "proto3";
message MsgDrone {
	int32 id = 1;
	int32 sim_step = 2;

	repeated double pos = 3;
	repeated double vel = 4;
	repeated double att_quat = 5;
	repeated double att_rate = 6;

	//video = 4;
}

